peer/core.yaml

```diff
@@ -335,6 +335,13 @@ ledger:
       # Deploy chaincodes into the genesis block
       chaincodes:
 
+        noop_syscc:
+          path: github.com/hyperledger/fabric/core/system_chaincode/noop
+          type: GOLANG
+          constructor:
+            args:
+
+
```

Comment:
Do we want to have this enabled by default?  It seems that could be construed as a security risk since any transaction could be queried.  Or would all transactions committed to the ledger be considered public info anyway?

Either way, i think it would be more palatable if this was disabled by default.  Thoughts @muralisrini @tamasblummer